kotlin-react/src/main/kotlin/react/Imports.kt

```diff
@@ -110,20 +110,28 @@ external val Profiler: RClass<ProfilerProps>
 external fun <T : Any> useState(): StateInstance<T?>
 
 external fun <T> useState(
-    initValue: T,
+    initialValue: T,
 ): StateInstance<T>
 
 external fun <T> useState(
-    valueInitializer: () -> T,
+    initializer: () -> T,
 ): StateInstance<T>
 
 // Reducer Hook (16.8+)
-@JsName("useReducer")
-external fun <S, A> rawUseReducer(
+external fun <S, A, I> useReducer(
+    reducer: RReducer<S, A>,
+    initializerArg: I,
```

Comment:
`initialArg` in original declarations?

Reply:
I have used names from [Definitely Typed](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L988), I like their naming more because of clarity